shell/src/main/java/alluxio/cli/util/DistributedCommandUtil.java

```diff
@@ -84,13 +84,10 @@ public static void getDetailedCmdStatus(
       printKeyWord = "processed";
     }
 
+    System.out.format("Get command status information below: %n");
```

Comment:
System.out.format does not print anything,  i think you mean println?

Reply:
it actually print out a message, String.format doesn't print anything

Reply:
It prints out the content on the screen actually.

Reply:
changed to println, will wait for the test to pass